compiler/il/OMRILOpCodesEnum.hpp

```diff
@@ -534,6 +534,7 @@
    vcalli,     // indirect call returning a vector
    vternary,   // vector ternary operator
    v2v,        // vector to vector conversion. preserves bit pattern (noop), only changes datatype
+   vl2vd,      // vector to vector conversion. converts each long element to double
    vconst,     // vector constant
```

Comment:
Another good find. I made the opcode typed for two reasons:

1) there have been a lot of discussions about pros and cons of typeless opcodes. There is a possibility that we make them typed again
2) currently we need a very safe change and making vl2vd typed is much safer solution for now